source/_data/themes.yml

```
@@ -3076,3 +3076,12 @@
     - hexo-theme
     - simple
     - clean
+- name: leo
+  description: A simple, elegant, clear and responsive theme for Hexo.
+  link: https://github.com/lixuetaoleo/hexo-theme-leo
+  preview: https://lixuetaoleo.github.io/
+  tags:
+    - simple
+    - elegant
+    - clear
+    - responsive
```

```suggestion
    - responsive

```

Would you mind adding a new line at the end of the file?